tuneme/styles/tuneme/_base.scss

```diff
@@ -55,7 +55,6 @@ input[type="checkbox"],
 input[type="radio"]  {
   display: inline-block;
   margin-left: 10px;
```

Comment:
@moh-moola please remove 
margin-left: 10px;

Reply:
On the below selector:
.competition input[type="checkbox"], 
.yourwords input[type="checkbox"]
 Also remove margin-right: 10px

Reply:
Please change this to display: initial;
input[type="checkbox"] + label,
input[type="radio"] + label  {
   display: inline-block;
}

And check no side effect across forms.

Reply:
.competition input[type="checkbox"],
.yourwords input[type="checkbox"] selectors are not applicable/present in tuneme